obstacle_stop_planner_refine/src/debug_marker.cpp

```diff
@@ -314,249 +262,199 @@ visualization_msgs::msg::MarkerArray ObstacleStopPlannerDebugNode::makeVisualiza
+      "map",
+      current_time,
+      "stop_obstacle_text",
+      0,
+      visualization_msgs::msg::Marker::TEXT_VIEW_FACING,
+      autoware_utils::createMarkerScale(0.0, 0.0, 1.0),
+      autoware_utils::createMarkerColor(1.0, 1.0, 1.0, 0.999));
+
     marker.pose.position = *stop_obstacle_point_ptr_;
     marker.pose.position.z += 2.0;
     marker.pose.orientation.x = 0.0;
     marker.pose.orientation.y = 0.0;
     marker.pose.orientation.z = 0.0;
     marker.pose.orientation.w = 1.0;
-    marker.scale.x = 0.0;
-    marker.scale.y = 0.0;
-    marker.scale.z = 1.0;
-    marker.color.a = 0.999;  // Don't forget to set the alpha!
-    marker.color.r = 1.0;
-    marker.color.g = 1.0;
-    marker.color.b = 1.0;
     marker.text = "!";
+
     msg.markers.push_back(marker);
   }
 
   if (slow_down_obstacle_point_ptr_ != nullptr) {
-    visualization_msgs::msg::Marker marker;
-    marker.header.frame_id = "map";
-    marker.header.stamp = current_time;
-    marker.ns = "slow_down_obstacle_point";
-    marker.id = 0;
-    marker.lifetime = rclcpp::Duration::from_seconds(0.5);
-    marker.type = visualization_msgs::msg::Marker::SPHERE;
-    marker.action = visualization_msgs::msg::Marker::ADD;
+    auto marker = autoware_utils::createDefaultMarker(
+      "map",
+      current_time,
+      "slow_down_obstacle_point",
+      0,
+      visualization_msgs::msg::Marker::SPHERE,
+      autoware_utils::createMarkerScale(0.25, 0.25, 0.25),
+      autoware_utils::createMarkerColor(1.0, 0.0, 0.0, 0.999));
+
     marker.pose.position = *slow_down_obstacle_point_ptr_;
     marker.pose.orientation.x = 0.0;
     marker.pose.orientation.y = 0.0;
     marker.pose.orientation.z = 0.0;
     marker.pose.orientation.w = 1.0;
-    marker.scale.x = 0.25;
-    marker.scale.y = 0.25;
-    marker.scale.z = 0.25;
-    marker.color.a = 0.999;  // Don't forget to set the alpha!
-    marker.color.r = 1.0;
-    marker.color.g = 0.0;
-    marker.color.b = 0.0;
+
     msg.markers.push_back(marker);
   }
 
   if (slow_down_obstacle_point_ptr_ != nullptr) {
-    visualization_msgs::msg::Marker marker;
-    marker.header.frame_id = "map";
-    marker.header.stamp = current_time;
-    marker.ns = "slow_down_obstacle_text";
-    marker.id = 0;
-    marker.lifetime = rclcpp::Duration::from_seconds(0.5);
-    marker.type = visualization_msgs::msg::Marker::TEXT_VIEW_FACING;
-    marker.action = visualization_msgs::msg::Marker::ADD;
+    auto marker = autoware_utils::createDefaultMarker(
+      "map",
+      current_time,
+      "slow_down_obstacle_text",
+      0,
+      visualization_msgs::msg::Marker::TEXT_VIEW_FACING,
+      autoware_utils::createMarkerScale(0.0, 0.0, 1.0),
+      autoware_utils::createMarkerColor(1.0, 1.0, 1.0, 0.999));
+
     marker.pose.position = *slow_down_obstacle_point_ptr_;
     marker.pose.position.z += 2.0;
     marker.pose.orientation.x = 0.0;
     marker.pose.orientation.y = 0.0;
     marker.pose.orientation.z = 0.0;
     marker.pose.orientation.w = 1.0;
```

Comment:
Probably can be removed?

Reply:
Thanks for comment. I removed unnecessary code at 2a7c51a